core-java-modules/core-java-concurrency-advanced/src/main/java/com/baeldung/concurrent/countdownlatch/Worker.java

```diff
@@ -15,7 +20,7 @@ public class Worker implements Runnable {
     @Override
     public void run() {
         // Do some work
-        System.out.println("Doing some logic");
```

Comment:
Is this change relevant to the task? As per description: "We need to ensure that our log statements generated **during the builds** ".